src/style.js

```diff
@@ -172,9 +172,17 @@ export function toggle(element, opt_display) {
  * @return {string}
  */
 export function px(value) {
-  return value + 'px';
+  return `${value}px`;
 }
 
+/**
+ * Returns a degree value.
+ * @param {number} value
+ * @return {string}
+ */
+export function deg(value) {
```

Comment:
Are you using it in your coming PRs? Otherwise, I don't think it's worth the abstraction

Reply:
I think the readability benefit, plus AMP consistency make it worth keeping.

Reply:
Leaving it for readability plus AMP consistency. Also highly likely that we will use it later for other animations.

Reply:
Also turns typos into compilation errors instead of just dropping the style, which is a good thing